tests/fixtures/marine/marine_fixtures.py

```diff
@@ -1,93 +1,266 @@
 import os
 from ipaddress import IPv4Address
-from random import getrandbits, randint
+from random import getrandbits, randint, choices
+from typing import TypedDict, List
+import string
 
 import pytest
-from pypacker.layer12 import ethernet
-from pypacker.layer3 import ip
+from pypacker.layer12 import ethernet, arp
+from pypacker.layer3 import ip, icmp
 from pypacker.layer4 import tcp, udp
+from pypacker.layer567 import dns, http
 
 from marine import Marine
 
 
+class HttpDescription(TypedDict):
+    http_type: str
+    uri: str
+    version: str
+    host: str
+    body: str
+
+
 def create_random_ip() -> str:
     return str(IPv4Address(getrandbits(32)))
 
 
 def create_random_mac() -> str:
-    return '00:00:00:%02x:%02x:%02x' % (randint(0, 255), randint(0, 255), randint(0, 255))
+    return "00:00:00:%02x:%02x:%02x" % (
+        randint(0, 255),
+        randint(0, 255),
+        randint(0, 255),
+    )
+
+
+def create_random_string(length: int) -> str:
+    return "".join(choices(string.ascii_lowercase, k=length))
+
+
+def create_random_url(length: int = 8) -> str:
+    return "www.%s.com" % (create_random_string(length))
+
+
+def create_random_http_get() -> HttpDescription:
```

Comment:
I don't think there's significance in stating this is specifically `get`. If you think there is, then consider adding at least another type to justify the specificity.